keyboards/skyloong/gk61/v1/info.json

```diff
@@ -0,0 +1,133 @@
+{
+    "keyboard_name": "GK61",
+    "maintainer": "JZ-Skyloong",
+
+    "features": {
+        "command": false,
+        "console": false,
+        "rgb_matrix": true,
+        "encoder": false
```

Comment:
```suggestion
        "rgb_matrix": true
```

Reply:
It is expected that encoder will be supported in the V2 keyboard and would like to keep this option to reflect version differences. Do you think that's appropriate? @zvecr

Reply:
No, the key should be removed as setting it to false serves no purpose. If and when a v2 is added, that will have its own info.json and that may have the key set to true.

Reply:
I see , thank you very much.